frameworks/hdfs/tools/docker-client/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:16.04
+
+RUN apt-get -y update && \
+    apt-get -y upgrade && \
+    apt-get -y install wget software-properties-common
+
+RUN wget https://downloads.mesosphere.com/hdfs/assets/hadoop-2.6.0-cdh5.9.1-dcos.tar.gz && \
+    tar xf hadoop-2.6.0-cdh5.9.1-dcos.tar.gz && \
+    rm hadoop-2.6.0-cdh5.9.1-dcos.tar.gz
+
+RUN apt-get update && \
+    apt-get install -y default-jre && \
+    apt-get install -y \
+    curl \
+    dnsutils \
+    less \
+    vim
+
+ENV HADOOP_VERSION hadoop-2.6.0-cdh5.9.1
```

Comment:
Let's use this to construct the HDFS download URL so that we don't have to change it in multiple places.